src/main.zig

```diff
@@ -616,7 +617,7 @@ fn buildOutputType(
     var subsystem: ?std.Target.SubSystem = null;
     var major_subsystem_version: ?u32 = null;
     var minor_subsystem_version: ?u32 = null;
-    var wasi_exec_model: ?wasi_libc.CRTFile = null;
+    var wasi_exec_model: ?[]const u8 = null;
```

Comment:
Hmm, I'd rather avoid passing strings instead of enums. How about we create an intermediate enum like you did before which then we can map to `wasi_libc.CRTFile` where required, and we can switch on in the linker rather than compare strings?

Reply:
yeah that's what actually I was thinking about first. Thanks for the suggestion! will fix accordingly.

Reply:
PTAL at https://github.com/ziglang/zig/pull/9178/commits/6769659dedde8b53999fda15b0125639cc451c23